docgen/templates/api.md

```diff
@@ -56,3 +56,16 @@ Status | Description
 {{range $k, $v := $m.Operation.Responses}}{{$k}} | {{GetText $v.Description}}
 {{end}}{{range $k, $v := $.Swagger.Responses}}{{$k}} | {{GetText $v.Description}}
 {{end}}{{end}}{{end}}
+
+## {{GetText "Annexe"}}
```

Comment:
I know this is correct in british english, but we generally follow american, so please spell it "annex" as a noun

Reply:
Done! I didn't know :)